flixel/tweens/FlxTween.hx

```diff
@@ -523,6 +567,39 @@ class FlxTween implements IFlxDestroyable
 		active = false;
 		_running = false;
 		finished = true;
+		
+		if (_thens != null && _thens.length > 0)
+		{
+			var then = _thens[0];
+			
+			_thens.splice(0, 1);
+			
+			if (then.delay <= 0)
```

Comment:
Right you were, it needs to call onEnd() again in the case that delay is <= 0 AND the tween is null, signifying it's a delay that should terminate instantly and run the next "then" command.